src/client/pages/Offer/Introduction/Sidebar/index.tsx

```diff
@@ -163,15 +163,8 @@ export const Sidebar: React.FC<SidebarProps> = ({
   )
 
   const handleRemoveCampaign = useCallback(async () => {
-    const { data: result } = await removeCampaignCode({
-      variables: { quoteCartId },
-    })
-    const hasError =
-      result?.quoteCart_removeCampaign.__typename === 'BasicError'
-
-    if (!hasError) {
-      CampaignCode.remove()
-    }
+    CampaignCode.remove()
+    await removeCampaignCode({ variables: { quoteCartId } })
```

Comment:
We were checking if the API worked before removing it from cookie storage. Wouldn't be a potencial problem not to check?

Reply:
I don't really see that. It's more of a bonus that we remember the code in the first place :)